Gems/MotionMatching/Code/Source/CsvSerializers.cpp

```diff
@@ -295,7 +311,12 @@ namespace EMotionFX::MotionMatching
                 transform.m_rotation = AZ::Quaternion::CreateFromMatrix3x3(rotationMatrix);
             }
 
-            pose.SetLocalSpaceTransform(jointIndex, transform);
+            switch (transformSpace)
+            {
+            case TRANSFORM_SPACE_LOCAL: { pose.SetLocalSpaceTransform(jointIndex, transform); break; }
```

Comment:
nitpick - not sure what the coding standards would say about this all being on one line (same for the other switch statements)

Reply:
This is a good point, it can make debugging these things harder too (if you select this block in Visual Studio and hit Ctrl-K, F, then you can format the selection and it'll automatically do the right thing)

Reply:
Applied this, will come with the next PR.

Reply:
Awesome thanks :+1: